extensions/amp-accordion/1.0/accordion.type.js

```diff
@@ -99,6 +100,7 @@ AccordionDef.ContentShimProps;
  *   toggleExpanded: (function(string)|undefined),
  *   animate: (boolean|undefined),
  *   prefix: (string),
+ *   experimentDisplayLocking: (boolean),
```

Comment:
fyi for singular types, we don't need parens. It looks like they are used this way throughout this file so no need to change.

Reply:
Thanks!